src/reader.rs

```diff
@@ -930,6 +1031,64 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_read_link_type_link() {
+        let kml_str = r#"<Link id="Some ID">
+            <href>/path/to/local/resource</href>
+            <refreshMode>onChange</refreshMode>
+            <refreshInterval>4</refreshInterval>
+            <viewRefreshMode>onStop</viewRefreshMode>
+            <viewRefreshTime>4</viewRefreshTime>
+            <viewBoundScale>1</viewBoundScale>
+            <viewFormat></viewFormat>
+        </Link>"#;
+
+        let mut attrs = HashMap::new();
+        attrs.insert("id".to_string(), "Some ID".to_string());
+
+        let l: Kml = kml_str.parse().unwrap();
+        assert_eq!(
+            l,
+            Kml::LinkTypeLink(LinkTypeLink {
+                href: Some("/path/to/local/resource".to_string()),
+                refresh_mode: Some(types::RefreshMode::OnChange),
+                view_refresh_mode: Some(types::ViewRefreshMode::OnStop),
+                view_format: Some(String::new()),
```

Comment:
In this test, `viewFormat` is an empty element, and this is treated as a value. Any desire to identify this and assign the optional String value to `None`?

Reply:
I think the test and behavior work as-is, since I would assume `None` would mean the tag isn't present at all